lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
I'm typing from mobile, but this might work:
```js
var createAccessor = embed_tokens(function() {
  return function_(AST_Accessor);
});
```

If not, I'll be back in a couple of hours and will take a look then.